specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -164,6 +165,201 @@ Consider at least the below topics but you do not need to cover those that are n
+  execution success rate - shows the success rate for all data job
+  [executions](https://github.com/vmware/versatile-data-kit/wiki/dictionary#data-job-execution)
+  in the last 14 days
+- data jobs with failing exectuions(last 24h) - the list can also be used to
+  navigate to individual job details
+- most recent failed executions(last 24h) - the list can also be used to
+  navigate to individual failed execution details
+
+**Data Jobs Page**
+
+![](images/jobs_page.png)
+
+Shows all existing data jobs and their properties
+- Job name
+- Description
+- Deployment Status
+- Last Execution End (UTC)
+- Last Execution Duration
+- Last Execution Status
+- Success rate
+- Schedule (in UTC)
+- Next run (UTC)
+- Last Deployed (UTC)
+- Last Deployed By
+- Notifications
+- Source
+- Logs
+- Details
+
+Jobs can be filtered by the following properites
+- Job name
+- Description
+- Deployment Status
+- Last Execution Status
+
+Individual data jobs can be enabled, disabled and executed from the appropriate
+buttons. Each job row in the table contains links to the source code, logs and
+details of the corresponding data job. A search box is also available.
+
+**Data Job Details Page**
+
+![](images/job_details.png)
+
+Shows the status, description, owner team, schedule (in UTC) and source
+location. Shows who gets notified on changing job status (deployment, success,
+user error, platform error). Shows the last 5 job executions, their status,
+execution time, duration, and links to logs. The job can be executed or deleted
+from the appropriate buttons. The data job executions details page can be viewed
+from the `Executions` tab.
+
+**Data Job Executions Details Page**
+
+![](images/job_executions.png)
+
+Shows a success/failure pie chart for the total executions of the data job.
+Shows a graph of job duration over time. Graph is zoomable and the time period
+can be adjusted. Shows execution history for the job. Executions history table
+has the following columns:
+- Status
+- Type
+- Duration
+- Start (UTC)
+- End (UTC)
+- ID
+- Version
+- Logs
+
+The execution history table can be filtered by
+- Status
+- Type
+- ID
+- Version
+
+Deployment details for an individual execution can be viewed using a button
+attached to the row.
+
+**Data Job Execution Deployment Details Pop-Up**
+
+![](images/job_execution_details.png)
+
+Shows the follwoing details about an individual execution
+- Version
+- Deployed by
+- Deployed at (UTC)
+- Schedule cron (UTC)
+- VDK version
+
+Shows the following resource information about an individual execution
+- CPU Limit
+- CPU Request
+- Memory Limit
+- Memory Request
+
+More info about k8s resource requests, resource limits and resource units.
+
+https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits
+
+https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes
+
+### User Journeys
+
+**Active development**
```

Comment:
nit: you may have in mind Active data jobs development? Active development seems quite broad of a subject to me